src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -89,8 +89,13 @@ object HeaderPlugin extends AutoPlugin {
         }(breakOut)
     }
 
-    val headers: SettingKey[Map[String, (Regex, String)]] =
-      settingKey("Header pattern and text by extension; empty by default")
+    val headerLicense: SettingKey[License] =
+      settingKey("The license to apply to files")
```

Comment:
I just had a good idea how to fix this. We simply add another implementation of the License trait:

```scala
final case class Custom(val text: String) extends License
```

and then:

```scala
headerLicense = Custom("My custom License text")
```